docs/wagtail-migrations.md

```diff
@@ -0,0 +1,81 @@
+# Wagtail and Django data migrations
+
+Django data migrations with Wagtail can be intereesting and challenging because programmatic editing of Wagtail pages [is difficult](https://github.com/torchbox/wagtail/issues/1101), pages have revisions, and StreamFields. This document is intended to describe ways we try to address these challenges in cfgov-refresh.
```

Comment:
Typo: "intereesting". Also is there a word missing before "StreamFields"? This sentence is awkward.